docs/specs/config.md

```diff
@@ -33,28 +36,21 @@
 [op.json](https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/.openpublishing.publish.config.json) [docfx.json](https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/docfx.json)
 
 ``` yml
-name: azure-documents
-product: Azure
-defaults: msdocs.yml
+name: Azure.azure-documents
+default: msdocs.yml
 basePath: azure
-content:
-- include:
-  - articles/**/*.{md,yml}
-  - bread/**/*.{md,yml}
-resource:
-- include:
-  - "**/*.{svg|png|jpg|jpeg|gif|svg}"
+include:
```

Comment:
Missing `files` before `include`

Reply:
If we needn't `content`/`resource` section, `include` and `exclude` can be extracted to top level.

---
In reply to: [182989589](https://github.com/dotnet/docfx/pull/2644#discussion_r182989589) [](ancestors = 182989589)

Reply:
Seems grouping them into a property has a better extensibility for future types, like `template`. Add a `content` property.

---
In reply to: [183260765](https://github.com/dotnet/docfx/pull/2644#discussion_r183260765) [](ancestors = 183260765,182989589)